src/istio/control/http/request_handler_impl.cc

```diff
@@ -48,14 +48,8 @@ CancelFunc RequestHandlerImpl::Check(CheckData* check_data,
                                      TransportCheckFunc transport,
                                      DoneFunc on_done) {
   ExtractRequestAttributes(check_data);
-
-  if (service_context_->client_context()->config().has_forward_attributes()) {
-    AttributesBuilder::ForwardAttributes(
-        service_context_->client_context()->config().forward_attributes(),
-        header_update);
-  } else {
-    header_update->RemoveIstioAttributes();
-  }
+  header_update->RemoveIstioAttributes();
```

Comment:
I think we should unconditionally remove it after extracting it, and not depend on presence of static forwarded attributes config.

Reply:
+1